ledger/sandbox-on-x/BUILD.bazel

```diff
@@ -194,11 +196,10 @@ da_scala_test_suite(
     ],
 )
 
-da_scala_test_suite(
+da_scala_library(
```

Comment:
Here we have a problem as we want to access test_suite modules in another module so we could inspect all test suites within the classpath.

Reply:
but now these tests are not run. this is WIP, right?

Reply:
they're not working, yes.